plugins/modules/files/sapcar.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: sapcar
+short_description: Manages SAPCAR from SAP
+version_added: "1.0.0"
+description:
+    - Provides support for unpacking sar/car files with the SAPCAR binary from SAP and pulling
+      information back into Ansible.
+options:
+  path:
+    description: The path to the SAR/CAR file.
+    required: true
+    type: path
+  dest:
+    description:
+      - The destination where SAPCAR extracts the SAR file. Missing folders will be created.
+        If this parameter is not provided it will unpack in the same folder as the SAR file.
+    required: false
+    type: path
+  binary_path:
+    description:
+      - The path to the SAPCAR binary e.g. /home/dummy/sapcar or https://myserver/SAPCAR.
+        If this parameter is not provided the module will try excute from application binary e.g /usr/local/bin.
+    required: false
+    type: path
+  signature:
+    description:
+      - If true the Signature will be extracted.
+    required: false
+    default: false
+    type: bool
+  manifest:
+    description:
+      - The Name of the manifest defaults to SIGNATURE.SMF.
+    required: false
+    default: "SIGNATURE.SMF"
+    type: str
+  remove:
+    description:
+      - If true the SAR/CAR file will be removed - !!!This should be used with caution!!!.
+    required: false
+    default: false
+    type: bool
+author:
+    - Rainer Leber (@RainerLeber)
+'''
+
+EXAMPLES = """
+# Extract SAR file - simple
+  - name: run sapcar
```

Comment:
```suggestion
  - name: Run sapcar
```
Task titles should start with an upper-case letter and should be somewhat descriptive. How about replacing the title with the text from the comment above?